projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta.py

```diff
@@ -11,9 +12,10 @@
 
 class IDataJobExecutor(abc.ABC):
     @abc.abstractmethod
-    def start_job(self, job_name: str, team_name: str):
+    def start_job(self, job_name: str, team_name: str, arguments: Dict):
```

Comment:
must be Optional as arugments can be missing (none) 
Set default 
```suggestion
    def start_job(self, job_name: str, team_name: str, arguments: Optional[Dict] = None):
```

Reply:
Done.